api/client/proto/authservice.proto

```diff
@@ -1053,6 +1073,19 @@ message CreateRecoveryCodesWithTokenResponse {
     repeated string RecoveryCodes = 1;
 }
 
+// CreatePrivilegeTokenRequest defines a request to create and get a user privilege token.
```

Comment:
I assume the privilege token exists so the user doesn't have to keep tapping the existing token in a small time window, correct?

Please explain why someone would create a token, what it does and how long it's expected to last.